configure.ac

```diff
@@ -1283,6 +1283,79 @@ else
     PAC_APPEND_FLAG([-L${with_openpa_prefix}/lib],[WRAPPER_LDFLAGS])
 fi
 
+# Izem
+
+AC_ARG_ENABLE(izem,
+        [AC_HELP_STRING([--enable-izem],[Enable using the Izem library])],,
+        [enable_izem=no])
```

Comment:
Do we need a separate `--enable-izem` call?  How is this different from `--without-zm-prefix`?

Reply:
the intention was that `--enable-izem` would "use" izem features (currently only locks) and `--with-zm-prefix` to control the izem dependency as a whole. The next step was to add `--enable-izem-xxx` that would enable/disable selective features. For example `--enable-izem-queues` and `--disable-izem-locks` would result in a Pthread mutex + izem queues combination. 

The right thing to do here I think is to rename `--enable-izem` to `--enable-izem-locks`, since it was the intent. Does that make sense?

Reply:
Yeah, that would be better.  You might want to consider a comma-separated listed of features, rather than an `enable/disable` option for each feature.